WordPress/gutenberg
# Cover Block: Fixed background causes image to be zoomed in on Safari browser on iPad

## What?

I'm borrowing heavily from @taipeicoder's issue description here: https://github.com/WordPress/gutenberg/issues/47726

There is an issue with the Cover Block in Safari on iPad. If you set a Cover Block to a "Fixed background", iPad Safari zooms in extremely close to the image and does not support the parallax effect. This was fixed in https://github.com/WordPress/gutenberg/pull/40087 However, that fix was undone in another PR https://github.com/WordPress/gutenberg/pull/40554

This PR puts the original fix in https://github.com/WordPress/gutenberg/pull/40087 back.

## Why?

This PR solves the problem described above.

## How?

By re-adding the fix.

## Testing Instructions





- You can test this PR by running Gutenberg and WordPress locally on Docker via [these instructions](https://github.com/WordPress/gutenberg/blob/605aeb0f4f7d2225120e498f95ae27b9f56d77a3/docs/contributors/code/getting-started-with-code-contribution.md).
- You'll also need xcode Simulator installed to simulate an iPad.
- First run Gutenberg `trunk` on Docker.
- Then, from your desktop browser, go to Gutenberg -> Demo on wp-admin.
- <img width="400" alt="step1" src="https://user-images.githubusercontent.com/140841/224440787-af7ecc7a-e75f-4864-8753-1f9cf6ec56ed.png">
- From there you can toggle the demo's Cover Block to "Fixed background"
- <img width="400" alt="step2" src="https://user-images.githubusercontent.com/140841/224440900-b3a58b93-dd8d-487c-a14e-a754c97f2fef.png">
- Publish that page and now visit it on you xcode Simulator iPad. It should look like this (it's gray because it's zoomed way into the image):
- <img width="400" alt="before" src="https://user-images.githubusercontent.com/140841/224441099-40e1ac4c-6341-4d74-b362-56afd89b0ca7.png">
- Now install this PR and refresh the page on your xcode Simulator iPad.
- With the fix it should look like this:
- <img width="400" alt="after" src="https://user-images.githubusercontent.com/140841/224441222-4552754b-f4d5-479b-a288-d446226d9d92.png">

Regression Test
Finally we need to test that this fix does not introduce any regressions to the PR that removed it. With this PR loaded, follow the testing instructions here https://github.com/WordPress/gutenberg/pull/40554 and make sure this PR didn't introduce a regression.




### Testing Instructions for Keyboard

NA

## Screenshots or screencast 

Before | After
--|--
<img width="941" alt="before" src="https://user-images.githubusercontent.com/140841/224440346-024b3add-dd5d-445e-9b21-b48a7d449b23.png">  | <img width="938" alt="after" src="https://user-images.githubusercontent.com/140841/224440363-5b1b274a-0d4b-4f14-bd89-8ed03addce49.png">
